app/services/intercom_service.rb

```diff
@@ -129,11 +129,22 @@ def self.intercom
     @intercom ||= Intercom::Client.new(token: EnvironmentCredentials.dig(:intercom, :intercom_access_token))
   end
 
+  MAX_RETRY_COUNT = 3
+
   def self.intercom_api(collection, verb, params)
     if Rails.env.development?
       Rails.logger.debug("Calling Intercom: #{collection}.#{verb}(#{params})")
     end
-    result = intercom.send(collection).send(verb, params)
+
+    retry_counts = 0
+
+    begin
+      result = intercom.send(collection).send(verb, params)
+    rescue
```

Comment:
this needs to be limited to just the networky errors we think are worthy of retry, possibly just  `Intercom::AuthenticationError`

otherwise it will retry needlessly 2x for other mistakes we might make in the future on that line that are less likely to be recoverable